### ISSUE TITLE
the loss value dosen't change using GCCA_loss

### ISSUE THREAD

**Issue by Yusin2Chen** [arminarj/DeepGCCA-pytorch#1]
When I use GCCA_loss to train my network, the loss always keeps the same value.
If you can give me some suggestions, I am very grateful.

**Comment by arminarj** [comment]
Hi @cyx669521, 
Thanks for your interest in this repo. But it seems the loss objective is not always working and I am still working to understand its bottleneck and conditions.
Do you want to share your model that we can analyze it step by step together?
Additionally, feel free to commit your ideas to the repo.
